docs/faq/proof-of-stake/general.md

```diff
@@ -74,11 +74,9 @@ It also reduces latency between the wallet and network which can reduce the chan
 
 #### 7. What happens to my funds when I buy a ticket? 
 
-Funds used to purchase tickets are locked until the ticket is selected to vote or it expires. They are unable to leave your wallet and will appear under the Locked heading of your wallet. If your ticket is selected to vote, you will be refunded the total amount paid for the ticket and given the PoS reward.
+Funds used to purchase tickets are locked until your ticket is selected to vote or expires; tickets expire if they aren't selected to vote after 40960 blocks (about 4 months). If your ticket does successfully vote when called, you will receive the PoS reward. 
 
-The only costs for using this system are the fees you set. The txfee will be charged for the transaction of your ticket across the network. The ticketfee will be charged for incorporation into the ticket pool, and is paid to the PoW miners. If your ticket does not get mined, there is no charge as the transaction never made it onto the network.
-
-In case your ticket isn't selected to vote after 40960 blocks (about 4 months), the system will revoke your ticket and the decred you paid for it (minus fees) will be transferred back into your wallet. 
```

Comment:
Why have you deleted this line?

Reply:
I basically just rewrote this whole answer to make it flow better, so just deleted original text. I believe I've included all the relevant info from deleted text.